tutorial/02 - Styling and Theming.ipynb

```diff
@@ -885,7 +881,7 @@
    "source": [
     "# Axes\n",
     "\n",
-    "Next we will take a look at [stlying of Axes](https://bokeh.pydata.org/en/latest/docs/user_guide/styling.html#axes).\n",
+    "Next,we will take a look at the [styling of Axes](https://bokeh.pydata.org/en/latest/docs/user_guide/styling.html#axes).\n",
```

Comment:
missing space after comma, but again I don't think a comma here is necessary. I also don't think the added "the" is necessary..